src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -131,6 +131,7 @@ Microsoft.CodeAnalysis.OperationKind.VariableDeclaration = 1030 -> Microsoft.Cod
 Microsoft.CodeAnalysis.OperationKind.VariableDeclarationStatement = 3 -> Microsoft.CodeAnalysis.OperationKind
 Microsoft.CodeAnalysis.OperationKind.YieldBreakStatement = 12 -> Microsoft.CodeAnalysis.OperationKind
 Microsoft.CodeAnalysis.OperationKind.YieldReturnStatement = 16 -> Microsoft.CodeAnalysis.OperationKind
+Microsoft.CodeAnalysis.RefKind.In = 3 -> Microsoft.CodeAnalysis.RefKind
 Microsoft.CodeAnalysis.RefKind.RefReadOnly = 3 -> Microsoft.CodeAnalysis.RefKind
```

Comment:
Should RefKind.RefReadonly be removed? Or is that covered by Vlad's PR?

Reply:
Nevermind, we need it for returns...

---
In reply to: [141945392](https://github.com/dotnet/roslyn/pull/22424#discussion_r141945392) [](ancestors = 141945392)